bootloaders/encrypted/hello_serial.c

```diff
@@ -6,8 +6,10 @@
 
 #include <stdio.h>
 #include "pico/stdlib.h"
+#include "hardware/sync.h"
 
 int main() {
+    restore_interrupts_from_disabled(0);
```

Comment:
i was gonna say we should explicitly call enable IRQs as the parameter here is platform specific and thus confusing; I will add an issue to make an `enable_interrupts()` method in the SDK

Reply:
https://github.com/raspberrypi/pico-sdk/issues/2262